.github/workflows/nightly.yml

```diff
@@ -2009,173 +2009,10 @@ jobs:
       testrun: ${{ needs.prepare-workflow.outputs.testrun }}
       salt-version: "${{ needs.prepare-workflow.outputs.salt-version }}"
       cache-prefix: ${{ needs.prepare-workflow.outputs.cache-seed }}|3.10.14
-      skip-code-coverage: false
+      skip-code-coverage: true
       workflow-slug: nightly
       default-timeout: 360
 
-  combine-all-code-coverage:
-    name: Combine Code Coverage
-    if: ${{ fromJSON(needs.prepare-workflow.outputs.testrun)['skip_code_coverage'] == false }}
```

Comment:
Can we leave this step here and just set this to False?

Reply:
:rofl All I did was change the jinja defaults to True for skip-code-coverage, the nightly.yml and scheduled.yml are generated files

Can dig into how that are generated, but betting that is getting deleted since skipping is set to true and hence not generated.

What would you like to do here